File: Ruby/json.rb
require 'json'

jsonStr = '{"name":"Matthew Wyskiel","school":"Christian Heritage School","skills":["programming","singing","the Rock Band video game"]}'
jsonObj = JSON.parse(jsonStr)

name = jsonObj["name"]
school = jsonObj["school"]
skills = jsonObj["skills"]

puts "Hello, my name is #{name}, I go to #{school}, and I am good at #{skills}"

File: JavaScript/json.js
var json = '{"name":"Matthew Wyskiel","school":"Christian Heritage School","skills":["programming","singing","the Rock Band video game"]}';
var obj = JSON.parse(json);

console.log("Hello, my name is " + obj.name + " and I go to " + obj.school + " and I am good at: " + obj.skills);
